internal/cli/atlas/backup/exports/buckets/create.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package buckets
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli/require"
+	"github.com/mongodb/mongodb-atlas-cli/internal/config"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/internal/store"
+	"github.com/mongodb/mongodb-atlas-cli/internal/usage"
+	"github.com/spf13/cobra"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+type CreateOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	iamRoleID     string
+	bucketName    string
+	cloudProvider string
+	store         store.ExportBucketsCreator
+}
+
+func (opts *CreateOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+var createTemplate = "Bucket '{{.BucketName}}' created.\n"
```

Comment:
[nit] tou are not creating a bucket per se, I'd wait for docs review but it feels to me this needs a better language

Reply:
You're right. I will pass the PR to review and we'll see what's their opinion.

Reply:
@sarahsimpers You have to have an AWS bucket before running this command. After using it, Atlas connects and configures the bucket for exporting by creating some directory and small files there.